### DISCUSSION TITLE
How to download jaxlib0.1.68 cuda version?

### DISCUSSION THREAD

**Discussion by EchoTHChen** [jax-ml/jax#13709] [General] [answered]
recently I need jaxlib old version such as 0.1.68+cuda110. I try to `pip install --upgrade jaxlib==0.1.68+cuda110 -f https://storage.googleapis.com/jax-releases/jax_releases.html` but it turns out:

ERROR: Could not find a version that satisfies the requirement jaxlib==0.1.68+cuda110 (from versions: 0.1.23, 0.1.29, 0.1.32, 0.1.40, 0.1.41, 0.1.42, 0.1.43, 0.1.44, 0.1.46, 0.1.50, 0.1.51, 0.1.52, 0.1.55, 0.1.56, 0.1.57, 0.1.58, 0.1.59, 0.1.60, 0.1.61, 0.1.62, 0.1.63, 0.1.64, 0.1.65, 0.1.66, 0.1.67, 0.1.68, 0.1.69) ERROR: No matching distribution found for jaxlib==0.1.68+cuda110

How can I install the jaxlib of cuda version from the official website?

**Comment by jakevdp** [comment, marked as answer]
You need to use the CUDA release URL: `https://storage.googleapis.com/jax-releases/jax_cuda_releases.html`

**Reply by EchoTHChen** [reply]
Thanks! It works for me.
